htdocs/admin/system/phpinfo.php

```diff
@@ -235,6 +235,20 @@
 print getResultColumn($name, $activatedExtensions, $loadedExtensions, $functions);
 print "</tr>";
 
+$functions = array();
+$name = "bz2";
+print "<tr>";
+print "<td>".$name."</td>";
+print getResultColumn($name, $activatedExtensions, $loadedExtensions, $functions);
+print "</tr>";
+
+$functions = array();
+$name = "bcmath";
```

Comment:
Do we need this php extension "bcmath" ?

Reply:
bcmath may be used in swiftmailer
<img width="620" height="330" alt="image" src="https://github.com/user-attachments/assets/dce69b63-1a80-44a1-8908-3c26198aaa96" />
I also seen the need in some external module...